src/main/java/com/chekanova/imagetool/validation/HexValidationUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.chekanova.imagetool.validation;
+
+import lombok.experimental.UtilityClass;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+import java.util.regex.Pattern;
+
+@UtilityClass
+public class HexValidationUtil {
+
+    private static final String INVALID_HEX = "hex.invalid";
+    private static final String HEX_PATTERN = "^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$";
+
+    private static final Pattern pattern = Pattern.compile(HEX_PATTERN);
+
+    public static void validateHex(String colorCode, RedirectAttributes attributes) {
+        if (!pattern.matcher(colorCode).matches()){
+            ResourceBundle bundle = ResourceBundle.getBundle("messages", Locale.ENGLISH);
+            String message = bundle.getString(HexValidationUtil.INVALID_HEX);
```

Comment:
```suggestion
            String message = bundle.getString(INVALID_HEX);
```
Do we really need to call variable using the class name?

Reply:
You are right, no need to write class name here